packages/core/utils/src/modules-sdk/medusa-internal-service.ts

```diff
@@ -252,12 +252,22 @@ export function MedusaInternalService<
       input: any | any[] | SelectorAndData | SelectorAndData[],
```

Comment:
thought: @shahednasser, do we have a reference for the internal module services? If not, would it make sense to have such that we can provide a link in the error message?

Reply:
Not a reference, but we have examples here: https://docs.medusajs.com/v2/advanced-development/modules/service-factory#generated-methods

We can generate a reference if we find it necessary, but the reason I didn't is I imagine the typings being confusing. We can alternatively have a document that provides more advanced / realistic examples